src/AppBundle/Resources/translations/admin-checklist.en.yml

```diff
@@ -55,6 +56,8 @@ checklistPage:
     giftsSummary: View gifts summary
     deputyExpensesSummary: View deputy expenses
     lifestyleResponses: View health and lifestyle responses
+    profDeputyCosts: View deputies responses
```

Comment:
Should this be 'deputy's', similar to line 125 below? (I'm aware the design has 'deputies').

Reply:
yes think you're right, good spot